doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,609 @@
+    Collecting ansys-mapdl-core
+    Downloading ansys_mapdl_core-0.68.1-py3-none-any.whl (26.9 MB)
+        ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 26.9/26.9 MB 37.3 MB/s eta 0:00:00
+    Collecting pexpect>=4.8.0
+    Using cached pexpect-4.9.0-py2.py3-none-any.whl (63 kB)
+    Collecting click>=8.1.3
+    ...
+
+You can test if this virtual environment is accessible from the compute nodes by
+running the following bash script ``test.sh``:
+
+.. code-block:: bash
+
+    #!/bin/bash
+    #SBATCH --job-name=myjob
+    #SBATCH --nodes=1
+    #SBATCH --ntasks-per-node=4
+    #SBATCH --time=01:00:00
+
+    # Commands to run
+    echo "Testing Python!"
+    source /home/user/.venv/bin/activate
+    python -c "from ansys.mapdl import core;print(f'PyMAPDL version {core.__version__} was successfully imported!')"
+
+using: 
+
+.. code-block:: console
+
+    user@machine:~$ srun test.sh
+
+This command might take around 1-2 minutes to complete depending on the amount of free
+resources available in the cluster.
+The console output should show:
+
+.. code-block:: text
+
+    Testing Python!
+    PyMAPDL version 0.68.1 was successfully imported!
+
+If you see an error in the output, visit :ref:`ref_hpc_troubleshooting`, especially :ref:`ref_python_venv_not_accesible`.
+
+Submit a PyMAPDL job
+--------------------
+
+To submit a PyMAPDL job, you need to create two files: a Python script
+with the PyMAPDL code and a bash script which activate the virtual environment
+and call the Python script.
+
+**Python script:** ``pymapdl_script.py``
+
+.. code-block:: python
+
+    from ansys.mapdl.core import launch_mapdl
+
+    # Number of processors must be lower than the
+    # number of CPUs allocated for the job.
+    mapdl = launch_mapdl(nproc=10)
+
+    mapdl.prep7()
+    n_proc = mapdl.get_value("ACTIVE", 0, "NUMCPU")
+    print(f"Number of CPUs: {n_proc}")
+
+    mapdl.exit()
+
+
+**Bash script:** ``job.sh``
+
+.. code-block:: bash
+
+    source /home/user/.venv/bin/activate
+    python pymapdl_script.py
+
+The simulation is then started using:
+
+.. code-block:: console
+
+    user@machine:~$ srun job.sh
+
+
+The bash file allow you to customize the environment before running the Python
+script (create new environment variables, move to different directories, do some printing
+to ensure your configuration is right, etc), however this file is not mandatory.
+You can avoid having the bash file ``job.sh`` if the virtual environment is activated,
+and you pass all the environment variables to the job:
+
+.. code-block:: console
+
+    user@machine:~$ source /home/user/.venv/bin/activate
+    (.venv) user@machine:~$ srun python pymapdl_script.py --export=ALL
+
+
+The ``--export=ALL`` argument might not be needed, depending on the cluster configuration.
+Furthermore, you can omit the ``python`` call in the preceding command, if there is
+the Python shebang (``#!/usr/bin/python3``) in the ``pymapdl_script.py`` script first line.
```

Comment:
@germa89 the python shebang would make this incompatible with a virtual environment, right?

Reply:
mmhhh... it does work with a venv....

Reply:
I think if i use ``#! python3`` it should work

Reply:
I didnt see any example on having ``python` only as sheabag... hence I'm not using it.

And since it works with the VENV, I'm keeping it as it is.